latest/base.yml

```diff
@@ -72,17 +72,17 @@ docker_images:
   # renovate: datasource=docker depName=zookeeper
   zookeeper: '3.8.1'
   # renovate: datasource=git-tags depName=https://opendev.org/zuul/zuul.git
-  zuul_client: '8.2.0'
+  zuul_client: '8.3.1'
   # renovate: datasource=git-tags depName=https://opendev.org/zuul/zuul.git
-  zuul_executor: '8.2.0'
+  zuul_executor: '8.3.1'
   # renovate: datasource=git-tags depName=https://opendev.org/zuul/zuul.git
-  zuul_nodepool_builder: '8.2.0'
+  zuul_nodepool_builder: '8.3.1'
```

Comment:
This is wrong, nodepool is a different project with its own set of tags https://opendev.org/zuul/nodepool

Also why is the ".git" extension needed for the data source?

Reply:
Not sure if .git is really required.

Reply:
This is still the wrong tag, I think you changed the wrong renovate comments.

Reply:
Ah forgot the node pool builder.